pkg/network/network.go

```diff
@@ -56,6 +61,10 @@ type Config struct {
 
 	// DefaultClusterIngressClass specifies the default ClusterIngress class.
 	DefaultClusterIngressClass string
+
+	// RouteTemplate is the golang text template to use to generate the
+	// Route's URL for the Service.
+	RouteTemplate string
```

Comment:
I'm wondering if we could/should make this part of the Route itself as well (and default to the configured value). That'd make it possible to throw whatever domain one might want in there.

Reply:
I think allowing a Route to have its own value is a good idea, but I'd still want to have a default global one that's configurable. We could do the Route-level one in a follow-on PR.